src/ran/exception/MissingDescriptionException.java

```diff
@@ -0,0 +1,25 @@
+package ran.exception;
+
+import ran.task.TaskType;
+
+public class MissingDescriptionException extends RanException {
+    private TaskType type;
+
+    public MissingDescriptionException (TaskType type) {
+        this.type = type;
+    }
+
+    public String getTypeString () {
+        switch (type) {
+        case TODO: 
+            return "todo";
+        case DEADLINE:
+            return "deadline";
+        case EVENT:
+            return "event";
+        case UNDEFINED:
+        default:
```

Comment:
Are the switch cases meant to be empty? If so, do indicate a comment so that others know that it is an intended behaviour and not a mistake